Snowflake.Data/Core/ReusableChunkParser.cs

```diff
@@ -123,6 +123,9 @@ public async Task ParseChunk(IResultChunk chunk)
                                     break;
                                 case -1:
                                     throw new SnowflakeDbException(SFError.INTERNAL_ERROR, $"Unexpected end of stream in escape sequence");
+                                default:

```

Comment:
Do we need to handle other escape case as well? Make sure we are good with all these cases: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/strings/#string-escape-sequences

Reply:
it already cover all escape cases.